### ISSUE TITLE
run_analysis not able to process VOT2015 results in octave

### ISSUE THREAD

**Issue by bjanssen** [votchallenge/vot-toolkit#90]
The following error is produced
```
          Processing selector sequence_bag ...
            Processing tracker FragTrack ...
error: trackers_raw_scores_selector: A(I) = X: X must have the same size as I
error: called from
    analyze_ranks>trackers_raw_scores_selector at line 199 column 30
    analyze_ranks>trackers_ranking at line 113 column 79
    analyze_ranks at line 79 column 34
    report_cache at line 26 column 12
    report_ranking at line 85 column 12
    report_article at line 74 column 32
    run_analysis at line 16 column 1
```
many trackers, but not all, give the same problem.

Ubuntu 16.04
Octave 4.0.2
trackers: http://box.vicos.si/vot/vot2015_trackers.zip
results: http://box.vicos.si/vot/vot2015_results.zip
stack: 'vot2015'

Reproduce:
- create a workspace: `workspace_create`
- extract results and trackers zip files in your workspace
- edit `run_analysis.m` script:
  - comment the line `%error('Analysis not configured! Please edit run_analysis.m file.'); % Remove this line after proper configuration`
  - modify the line containing the `tracker_list` command to read: `trackers = tracker_list('reference.txt');`
- run the `run_analysis` script

**Comment by lukacu** [comment]
Thank you for the report, but please write more details so that we can reproduce the error, e.g. platform, which script did you run, etc.

**Comment by bjanssen** [comment]
@lukacu I modified the description, in case `reference.txt` is used (from the trackers zip file), the first error that occurs is generated by `FragTrack`.

Before the fatal error a lot of messages like this get generated:
```
            Processing tracker FoT ...
            Warning: less than 25 samples when comparing trackers. Cannot reject hypothesis
warning: division by zero
            Warning: less than 25 samples when comparing trackers. Cannot reject hypothesis
            Warning: less than 25 samples when comparing trackers. Cannot reject hypothesis
            Warning: less than 25 samples when comparing trackers. Cannot reject hypothesis
warning: division by zero
            Warning: less than 25 samples when comparing trackers. Cannot reject hypothesis
            Warning: less than 25 samples when comparing trackers. Cannot reject hypothesis
            Warning: less than 25 samples when comparing trackers. Cannot reject hypothesis
            Warning: less than 25 samples when comparing trackers. Cannot reject hypothesis
```

**Comment by bjanssen** [comment]
Should be fixed by PR #91 
Warnings are still there though.

**Comment by lukacu** [comment]
Thanks for checking it, I still have a few questions before I merge it because it looks like the problem is deeper so I will investigate if the PR is enough. Regrading warnings: this is unfortunately normal since some trackers fail so often that there are hardly any useful frames left. We may want to remove the warnings because they are confusing.

**Comment by bjanssen** [comment]
A similar issue appears to happen a bit later:
```
        Tracker NCC
error: hist: subscript indices must be either positive integers less than 2^31 or logicals
error: called from
    hist at line 150 column 11
    estimate_expected_overlap at line 47 column 12
    analyze_expected_overlap at line 84 column 67
    report_cache at line 26 column 12
    report_expected_overlap at line 61 column 12
    report_article at line 82 column 52
    run_analysis at line 16 column 1
```
where `context.sources = [](0x00)` like all other elements of the `context` cell-array.

**Comment by lukacu** [comment]
This is strange, I will have to investigate it myself. I will first try to reproduce it in Matlab to see if it is a common problem or an Octave-specific one.

**Comment by bjanssen** [comment]
This also reproduces on OSX with octave 4.0.0

**Comment by lukacu** [comment]
Ok, I can confirm that it is an Octave-specific bug, it does not happen in Matlab, but occurs in Octave as you have described. I will investigate further because this means that there are some differences between the languages.

**Comment by lukacu** [comment]
Ok, I have accepted fix in pull request #91 , there is an explanation regarding the underlying reasons in the comment of the request. Closing the issue.
